### ISSUE TITLE
[task](phase1): our phase 1 models need to be retrained

### ISSUE THREAD

**Issue by kyrieb-ekat** [DDMAL/mothra#278]
As we've progressed in the project, the precise type of ground truth we need has been narrowed in on, and the bumps in the [pipeline](https://github.com/DDMAL/mothra/discussions/95) have revealed some target areas that should improve phase 1 model performance (model 1: music + text; model 2: staffline bboxes around individual lines on an ink separated page). The wonderful @JoyfulGen has made the ground truth for this; questions about the ground truth and their location(s) should be directed to her. 

@lizima trained the v1 phase1 models, and should be consulted on what steps she previously took on training settings and fine-tuning (if any); she used YOLOv11, but I am amenable if in the course of looking at performance an other yolo model might be worth swapping in. 

The staffline model has the most change: we have had a shift in our method for what type of images get passed into the staffline branch (see project paths in the discussion here). Where previously we send a whole image on, performance was bad enough we shifted gears to re-utilizing Calvo and Paco for training a model and separating ink from stafflines from the rest of the page. It is this shift, to labeling background-less boxes that this retraining will primarily address: all backgrounds have been removed, labels applied, etc. 

This is a long-term ask, and higher priority than the phase2 neume-splitting model that will also be trained and used (one day...). I will link those issues here [https://github.com/DDMAL/Standalone-Interactive-Classifier/issues/100 and https://github.com/DDMAL/mothra-annotator/issues/23], and will copy the notes I have made so far on the neume-splitting experiments over when I've had a bit to gather my thoughts/track down the rest of my notes...
